cmake/CeleritasAddTest.cmake

```diff
@@ -379,6 +402,12 @@ function(celeritas_add_test SOURCE_FILE)
         file(MAKE_DIRECTORY "${_test_dir}")
       endif()
 
+      # Resource lock slows down overall test time but gives more accurate
+      # per-test time readings
+      # if(PARSE_GPU)
+      #   set_property(TEST ${_TEST_NAME}
+      #     PROPERTY RESOURCE_LOCK gpu)
+      # endif()
```

Comment:
Is this meant to be commented out?

Reply:
 Yes: uncommenting it slows down the overall test time in parallel but improves the timing statistics of the individual tests. I've refactored the script so that the resource lock is enabled for non-debug builds for accurate times, but disabled for debug so it should be clearer now.